### PR TITLE
Add support for action log messages

### PR DESCRIPTION
**Pull request by wallyworld**
Actions now have logs - add support for these in import/export.

### CODE REVIEW

File: action.go [hunk 1]
```diff
@@ -182,11 +247,32 @@ func importActionV1(source map[string]interface{}) (*action, error) {
...
+func importActionV1(source map[string]interface{}) (*action, error) {
+	fields, defaults := actionV1Fields()
+	return importAction(fields, defaults, 1, source)
+}
+
+func importActionV2(source map[string]interface{}) (*action, error) {
+	fields, defaults := actionV2Fields()
+	return importAction(fields, defaults, 2, source)
+}
+
```
Comment (babbageclunk) [inline action.go]: If you converted actions to use a map of fieldsFuncs then you wouldn't need this unsightly duplication. :)
Examples are cloudinstance or subnets.

Comment (wallyworld) [reply action.go]: done

**Comment by wallyworld** [comment]
$$merge$$

**Comment by wallyworld** [comment]
!!build!!

**Pull request merged**